includes/class-create-block-theme-api.php

```diff
@@ -105,7 +126,7 @@ function rest_get_readme_data( $request ) {
 			return new WP_REST_Response(
 				array(
 					'status'  => 'SUCCESS',
-					'message' => __( 'Readme file data retrieved.', 'create-block-theme' ),
+					'message' => __( 'Readme file data retrieved . ', 'create-block-theme' ),
```

Comment:
It seems like an auto-formatter was a bit overzealous with it's job. Will be correcting these.

Reply:
🤣